packages/components/src/components/pagination/_pagination.scss

```diff
@@ -42,6 +42,19 @@ $css--helpers: true;
     @include carbon--breakpoint('md') {
       overflow: initial;
     }
+
+    // mobile friendly pagination
+    @include carbon--breakpoint-down('md') {
+      .#{$prefix}--pagination__left > *,
+      .#{$prefix}--pagination__right > * {
+        display: none;
+      }
+
+      .#{$prefix}--pagination__items-count,
+      .#{$prefix}--pagination__controll-buttons {
+        display: initial;
```

Comment:
Seeing some issues between mobile and full-width

![Screen Shot 2020-07-28 at 10 50 34 AM](https://user-images.githubusercontent.com/11928039/88702579-5cf3c800-d0c0-11ea-9b04-0c781722c1b7.png)